packages/suite/src/components/suite/Labeling/components/WalletLabeling.tsx

```diff
@@ -2,13 +2,12 @@ import React from 'react';
 import { TrezorDevice } from '@suite-types';
 import { useTranslation } from '@suite-hooks/useTranslation';
 
-interface Props {
+interface WalletLabellingProps {
     device: TrezorDevice;
     useDeviceLabel?: boolean;
```

Comment:
would be nice to rename this prop to smthing that would not invoke it's hook. Maybe `shouldUseDeviceLabel`?

Reply:
done